File: src/main/java/net/cyberflame/entitysneak/EntitySneakPlugin.java
package net.cyberflame.entitysneak;

import net.cyberflame.entitysneak.listeners.SneakListener;
import org.bukkit.Bukkit;
import org.bukkit.plugin.java.JavaPlugin;

public class EntitySneakPlugin extends JavaPlugin
{
    public void onEnable()
    {
        Bukkit.getPluginManager().registerEvents(new SneakListener(), this);
        System.out.println("EntitySneak was enabled successfully!");
    }
}

File: src/main/java/net/cyberflame/entitysneak/EntitySneakPlugin.kt
package net.cyberflame.entitysneak

import org.bukkit.plugin.java.JavaPlugin
import org.bukkit.Bukkit
import net.cyberflame.entitysneak.listeners.SneakListener

class EntitySneakPlugin : JavaPlugin() {
    override fun onEnable() {
        Bukkit.getPluginManager().registerEvents(SneakListener(), this)
        println("EntitySneak was enabled successfully!")
    }
}